test/fixtures/app/fixture_cloud.js

```diff
@@ -1,16 +1,15 @@
 var nock = require('nock');
 var cloudReplies = {
-  headers : { 'Content-Type': 'application/json' },
-  act : function(){
-    return { ok : true, status : 'ok' };
+  headers: {'Content-Type': 'application/json'},
+  act: function (){
+    return {ok: true, status: 'ok'};
   }
 };
 
 module.exports = nock('https://apps.feedhenry.com')
-.filteringRequestBody(function() {
-  return '*';
-})
-.post('/cloud/somefunc')
-.reply(200, cloudReplies.act, cloudReplies.headers)
-.post('/some/custom/cloud/host')
-.reply(200, cloudReplies.act, cloudReplies.headers);
+  .filteringRequestBody(function (){
+    return '*';
+  })
+  .post('/cloud/somefunc')
+  .times(1)
```

Comment:
I don't think times is needed if you are only calling it once anyway.

Am I missing something?